extensions/control-plane/provision/provision-aws-s3/src/main/java/org/eclipse/edc/connector/provision/aws/s3/S3ConsumerResourceDefinitionGenerator.java

```diff
@@ -34,12 +34,15 @@ public class S3ConsumerResourceDefinitionGenerator implements ConsumerResourceDe
     public ResourceDefinition generate(DataRequest dataRequest, Policy policy) {
         if (dataRequest.getDataDestination().getProperty(S3BucketSchema.REGION) == null) {
             // FIXME generate region from policy engine
-            return S3BucketResourceDefinition.Builder.newInstance().id(randomUUID().toString()).bucketName(dataRequest.getDataDestination().getProperty(S3BucketSchema.BUCKET_NAME)).regionId(Region.US_EAST_1.id()).build();
+            return S3BucketResourceDefinition.Builder.newInstance().id(randomUUID().toString()).bucketName(dataRequest.getDataDestination().getProperty(S3BucketSchema.BUCKET_NAME))
```

Comment:
little refactor: the common part of the builder could be built before the `if` and used in both of the branches